src/lib_json/json_reader.cpp

```diff
@@ -21,25 +21,13 @@
 #include <utility>
 
 #include <cstdio>
-#if __cplusplus >= 201103L
-
-#if !defined(sscanf)
-#define sscanf std::sscanf
-#endif
-
-#endif //__cplusplus
 
 #if defined(_MSC_VER)
 #if !defined(_CRT_SECURE_CPP_OVERLOAD_STANDARD_NAMES)
 #define _CRT_SECURE_CPP_OVERLOAD_STANDARD_NAMES 1
```

Comment:
We're keeping this one, but I believe it's useless?  I think it's supposed to be defined before the `#include` of the CRT functions for which it selects behavior. And here we're defining it AFTER all of our `#include` statements.

https://docs.microsoft.com/en-us/cpp/c-runtime-library/secure-template-overloads?view=vs-2019

I don't have MSVC handy to verify this.

Reply:
Should I remove?